README.md

```diff
@@ -44,7 +56,7 @@ stack:
     url: https://youtu.be/-AuExRijtrA
   - title: "Starport Intro"
     duration: "4:12"
-    imgSrc: https://i.ytimg.com/vi/5RqAIE0b8Kw/hq720.jpg
+    imgSrc: https://i.imgur.com/JmyOl8n.jpeg
     url: https://youtu.be/5RqAIE0b8Kw
   - title: "CosmWasm & IBC - The New Horizon"
```

Comment:
**The video CosmWasm & IBC - The new Horizon** and **Blockchain Performance Measured Correctly** point to same video URL